content/strategy-goals/strategy/code-graph/index.md

```diff
@@ -14,11 +14,11 @@ Our mission is to **make discovering, learning, analyzing, and editing code easi
 
 Apart from the work each of our teams do for our mission, across Code Graph we're particularly focused now on improving the **discoverability** and **ease-of-use** around our [use cases](../index.md#use-cases). There are a few reasons why this is important:
 
-- Focusing on use cases ensures we're prioritizing areas where people get real value out of the product.
+- Focusing on use cases ensures we're prioritizing areas where people get real value out of the product
 - Improving discoverability increases feature adoption because it brings users into features at moments when the features would be most useful
 - Building cross-team capabilities helps us deliver features that are more than the sum of their parts (as opposed to shipping our org chart)
 
-Because this theme intentionally bridges team boundaries, we are especially mindful of following our [collaboration principles](../../../departments/product-engineering/process/cross-org-team-collab.md).
+Because this theme intentionally bridges team boundaries, we are especially mindful of following our [collaboration principles](../../../departments/product-engineering/process/cross-org-team-collab.md), which also includes details on how we collaborate with the [Growth & Integrations](../cloud/growth-and-integrations/index.md) team on our new user experience.
```

Comment:
```suggestion
Because this theme intentionally bridges team boundaries, we are especially mindful of following our [collaboration principles](../../../departments/product-engineering/process/cross-org-team-collab.md), which also includes details on how we collaborate with the [Growth & Integrations](../cloud/growth-and-integrations/index.md) team on our [new user experience](../../../departments/product-engineering/process/new-user-experience.md).
```